pkg/operation/cloudbotanist/packetbotanist/infrastructure.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packetbotanist
+
+import (
+	"github.com/gardener/gardener/pkg/operation/common"
+	"github.com/gardener/gardener/pkg/operation/terraformer"
+	"github.com/gardener/gardener/pkg/utils/flow"
+)
+
+const (
+	projectID = "projectID"
+	sshKey = "sshKey"
+)
+
+// DeployInfrastructure kicks off a Terraform job which deploys the infrastructure.
+func (b *PacketBotanist) DeployInfrastructure() error {
+	tf, err := b.NewShootTerraformer(common.TerraformerPurposeInfra)
+	if err != nil {
+		return err
+	}
+	return tf.
+		SetVariablesEnvironment(b.generateTerraformInfraVariablesEnvironment()).
+		InitializeWith(b.ChartInitializer("packet-infra", b.generateTerraformInfraConfig())).
+		Apply()
+}
+
+// DestroyInfrastructure kicks off a Terraform job which destroys the infrastructure.
+func (b *PacketBotanist) DestroyInfrastructure() error {
+        tf, err := b.NewShootTerraformer(common.TerraformerPurposeInfra)
+        if err != nil {
+                return err
+        }
+
+        var (
+                g = flow.NewGraph("Packet infrastructure destruction")
+
+                _ = g.Add(flow.Task{
```

Comment:
If your flow only consists out of this one single task (running the Terraformer) then I suggest to call it like here:  https://github.com/gardener/gardener/blob/master/pkg/operation/cloudbotanist/azurebotanist/infrastructure.go#L60-L67

Reply:
Will do, thanks.